lib/Refactoring/LocalRename.cpp

```diff
@@ -365,6 +367,10 @@ RenameLocs swift::ide::localRenameLocs(SourceFile *SF,
     }
   }
 
+  if (valueDecl == nullptr) {
+    return RenameLocs();
+  }
```

Comment:
Wouldn't a null `valueDecl` crash in `getRenameScope` above?

Reply:
True. Then I was on the wrong track, trying to debug this issue purely based on a Swift toolchain with no debug info because I couldn’t reproduce with a locally-built toolchain.